docs/Contributing/Configuration-files.md

```diff
@@ -1296,4 +1296,5 @@ List of saved scripts that can be run on all hosts.
 
 > **Note:** More settings are included in the [contributor documentation](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Configuration-for-contributors.md). It's possible, although not recommended, to configure these settings in the YAML configuration file.
 
+<meta name="pageOrderInSection" value="3500">
```

Comment:
Hey @eashaw, I see that we have this meta element in each page in the contributors doc. Is this required?

Reply:
@marko-lisica No, that value is not required for contributors' doc pages.

Reply:
Thanks!